features/support/hooks.js

```diff
@@ -81,12 +81,18 @@ Before({ timeout: 30 * 1000 }, async function () {
     truncateCollection(this.sdk, 'tests', 'events'),
   ]);
 
-  await this.sdk.query({
-    controller: 'admin',
-    action: 'loadFixtures',
-    refresh: 'false',
-    body: defaultFixtures,
-  });
+  try {
+    await this.sdk.query({
+      controller: 'admin',
+      action: 'loadFixtures',
+      refresh: 'false',
+      body: defaultFixtures,
+    });
+  }
+  catch (error) {
+    console.dir(error, { depth: 10 });
```

Comment:
is it normal to use a `console.dir`here ? is there no `debug` function like in Kuzzle ?

Reply:
No since it's in the functional test framework and I want to display the entire error content